inhibit/inhibit.go

```diff
@@ -117,13 +117,17 @@ func (ih *Inhibitor) Mutes(lset model.LabelSet) bool {
 
 	for _, r := range ih.rules {
 		if r.TargetMatchers.Match(lset) && r.hasEqual(lset) {
-			ih.marker.SetInhibited(fp, true)
+			// We set a fake alert ID for now so that we can
+			// distinguish AlertStateSuppressed between silenced
+			// and/or inhibited. In the future there will be unique
+			// identifiers for alerts.
+			ih.marker.SetInhibited(fp, "123")
```

Comment:
I put these semantics in place now even though we won't use them. This does lead to potential confusing behavior if we attempt to `SetInhibited` elsewhere before unique identifiers for alerts exist, but the symmetry with `SetSilenced` made sense to me.